multimodal/src/autogluon/multimodal/models/fusion/base.py

```diff
@@ -0,0 +1,67 @@
+import logging
+
+from torch import nn
+
+from ...constants import AUTOMM, LABEL
+
+logger = logging.getLogger(AUTOMM)
+
+
+class BaseMultimodalFusionModel(nn.Module):
```

Comment:
We can also optionally make it an abstract class.

Reply:
Which one is better: `AbstractModel ` or `BaseModel`?

Reply:
Abstract might sound better. I can update it.

Reply:
Also, consider aligning the file and class names, i.e., using the same key word.